src/components/AssetSearch/components/SearchTermAssetList.tsx

```diff
@@ -0,0 +1,54 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import type { Asset } from '@shapeshiftoss/types'
+import { useMemo } from 'react'
+import { selectAssetsSortedByName } from 'state/slices/common-selectors'
+import { useAppSelector } from 'state/store'
+
+import { filterAssetsBySearchTerm } from '../helpers/filterAssetsBySearchTerm/filterAssetsBySearchTerm'
+import { GroupedAssetList } from './GroupedAssetList/GroupedAssetList'
+
+export type SearchTermAssetListProps = {
+  isLoading?: boolean
```

Comment:
Shouldn't be optional as it's always drilled down, and same comment here re: not drilling it down
```suggestion
  isLoading: boolean
```